pkg/resources/network_policy_attachment_acceptance_test.go

```diff
@@ -49,6 +50,20 @@ func TestAcc_NetworkPolicyAttachment(t *testing.T) {
 			},
 		},
 	})
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
I have two remarks here:
1. We move away from Parallel tests globally. They were introduced by our predecessors without the proper setup. Please change to Test.
2. Please extract this to separate `func TestAcc_...`.

Reply:
Done!